server/check-validity.js

```diff
@@ -0,0 +1,10 @@
+function handleValidity (arr) {
+  for (let i = 0; i < arr.length; i++) {
+    if(!arr[i]) {
+      return false
+    }
+  }
+  return true
+}
+
+module.exports = handleValidity
```

Comment:
It's not clear what this is validating based on it's name. I recommend a rename to `allTruthy`. Or, better yet, delete this module and use `Array.prototype.every()` where it's needed.

```js
var things = [2, 3, 4, undefined];
var isValid = things.every(thing => !!thing);
```

https://stackoverflow.com/questions/784929/what-is-the-not-not-operator-in-javascript